test/evp_test.c

```diff
@@ -1969,6 +1969,9 @@ static int pbe_test_init(EVP_TEST *t, const char *alg)
         TEST_error("Unknown pbe algorithm %s", alg);
     }
```

Comment:
Could you add `return 0;` here please.  It's an existing problem but one that should be fixed.